### ISSUE TITLE
Create 3 x further snippets for specific locales

### ISSUE THREAD

**Issue by adamlofting** [mozilla/donate.mozilla.org#959]
This was talked about on the call, but I haven't seen a ticket and wanted to make sure it was filed.

Follow up task to #911

We have several locales we would like to serve more localised snippets. These are examples where the locales share a language, but not a currency, and we would like to serve up the fundraising ask in the most appropriate currency. These 3 locales are receiving significant levels of traffic.

E.g.

* `en-GB` - use same copy as en-US, but change currency to GBP
* `en-CA` - use same copy as en-US, but change currency to CAD
* `es-MX` - use same copy as es, but change currency to MXD

**Comment by adamlofting** [comment]
@ScottDowne This setup lives with MoCo team, but I'm assigning to you in case you know if this has been filed with them, or who needs to see it. If not, we can check with @valianttry on Monday.

**Comment by ScottDowne** [comment]
I have made an ask for en-CA and en-GB in https://bugzilla.mozilla.org/show_bug.cgi?id=1210193

es-MX has a small blocker in that prod needs to be pushed before we can request the snippet, but I noted that in the issue so they know it's coming.

**Comment by ScottDowne** [comment]
We should create an en-AU too!

**Comment by valianttry** [comment]
We are getting a ton of INR traffic that happen to use en-US -- is this an opportunity to serve an en-US snippet with INR currency perchance?

**Comment by ScottDowne** [comment]
@valianttry I would say new issue because it feels like a unique case.

**Comment by valianttry** [comment]
Sure, and i understand it might not be possible to get country data in order to serve INR -- and the MVP for this problem is solved by simply providing INR in our drop down for currency (https://github.com/mozilla/donate.mozilla.org/issues/915). I defer to snippet team + you guys on possibility of this. :hand:

**Comment by valianttry** [comment]
Also @adamlofting is it pretty easy to pinpoint when these kinds of changes occur? It would be so interesting and good to report out the (presumably positive ROI) results of doing this in these high volume locales. :moneybag:

**Comment by adamlofting** [comment]
> Also @adamlofting is it pretty easy to pinpoint when these kinds of changes occur? It would be so interesting and good to report out the (presumably positive ROI) results of doing this in these high volume locales. :moneybag:

If our theory is right, we'll see good conversion on these new locales and an increase in the channels that had a more mixed audience earlier.

> We are getting a ton of INR traffic that happen to use en-US -- is this an opportunity to serve an en-US snippet with INR currency perchance? 

We should check if snippet team can identify location as well as browser locale in this case. If we can, that would be great.

**Comment by ScottDowne** [comment]
Quick update for this. We have an INR snippet and an es-MX, but not en-* yet.

**Comment by ScottDowne** [comment]
I'm going to close this, and reopen a ticket that focuses en-IN (English speaking India) as that seems to not be straight forward as the rest. https://github.com/MozillaFoundation/Advocacy/issues/350
